python/ql/lib/semmle/python/dataflow/new/internal/DataFlowImplConsistency.qll

```diff
@@ -47,6 +47,10 @@ private module Input implements InputSig<PythonDataFlow> {
   predicate identityLocalStepExclude(Node n) {
     not exists(n.getLocation().getFile().getRelativePath())
   }
+
+  predicate multipleArgumentCallExclude(ArgumentNode arg, DataFlowCall call) {
+    isArgumentNode(arg, call, _)
```

Comment:
Maybe add a comment about why this check is excluded?

Reply:
The Python team (@RasmusWL) are already aware of this, and they will look into it follow-up.

Reply:
(Difference in Python 2 and Python 3 output, even when excluding stdlib :grimacing:)